packages/forms/src/PaymentsForm.tsx

```diff
@@ -89,11 +104,11 @@ export function PaymentsForm({
   });
 
   useEffect(() => {
-    localSetValue('token', token || _.first(TOKENS)?.address);
-
+    localSetValue('token', nativeWrappedToken);
```

Comment:
this needs to be an object that matches the Select's options, `TOKENS`

Reply:
Umm why? It's just setting the default value of TOKEN changes???

Reply:
Ah you're right